CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [\#130](https://github.com/Manta-Network/manta-rs/pull/130) Add the sage script and the hardcoded tests for the security of mds matrix
 
 ### Changed
+- [\#132](https://github.com/Manta-Network/manta-rs/pull/132) Simplify algebra APIs and removing ECC-specific design
```

Comment:
The order of changelog is inconsistent. Some are in ascending order, and some is descending order

Reply:
The PR number does not reflect the order in which they were merged, just the order in which they were created. We don't have to keep this consistent, it's more about being able to link certain changes to certain PRs.